mrmustard/math/backend_manager.py

```diff
@@ -573,6 +574,12 @@ def hermite_renormalized_diagonal(
         """
         return self._apply("hermite_renormalized_diagonal", (A, B, C, cutoffs))
 
+    def hermite_renormalized_diagonal_batch(
+        self, A: Tensor, B: Tensor, C: Tensor, cutoffs: Tuple[int]
+    ) -> Tensor:
+        r"""Same as hermite_renormalized_diagonal but works for a batch of different B's."""
```

Comment:
Would you mind copy/pasting the docstring of hermite renormalized here? So that if I want to learn about this function, I don't need to go and look at another one